packages/fields/src/types/CalendarDay/views/Field.js

```diff
@@ -1,41 +1,35 @@
 /** @jsx jsx */
 
 import { jsx } from '@emotion/core';
-import { getYear } from 'date-fns';
-import { Component } from 'react';
 
 import { FieldContainer, FieldLabel, FieldInput } from '@arch-ui/fields';
 import { TextDayPicker } from '@arch-ui/day-picker';
+import { Alert } from '@arch-ui/alert';
 
-export default class CalendarDayField extends Component {
-  handleSelectedChange = value => {
-    const { field, onChange } = this.props;
-    if (
-      value === null ||
-      (getYear(value).toString().length <= 4 &&
-        getYear(value) <= field.config.yearRangeTo &&
-        getYear(value) >= field.config.yearRangeFrom)
-    ) {
-      onChange(value);
-    }
-  };
+const CalendarDayField = ({ autoFocus, field, value, errors, onChange }) => {
+  const htmlID = `ks-input-${field.path}`;
 
-  render() {
-    const { autoFocus, field, value, errors } = this.props;
-    const htmlID = `ks-input-${field.path}`;
+  return (
+    <FieldContainer>
+      <FieldLabel htmlFor={htmlID} field={field} errors={errors} />
+      <FieldInput>
+        <TextDayPicker
+          id={`ks-daypicker-${field.path}`}
```

Comment:
```suggestion
          id={htmlID}
```

Reply:
`htmlID` is `ks-input-${field.path}`, though. A different value.

Reply:
Also looks like the tests look for that value (`ks-daypicker-`).

Reply:
They need to be the same because we have `htmlFor={htmlID}` and this need to match the id on the input.

My suggestion is to change `const htmlID = `ks-input-${field.path}`;` to match

Reply:
Ok, changed `htmlID`.